autoflake.py

```diff
@@ -608,7 +612,8 @@ def fix_code(source, additional_imports=None, expand_star_imports=False,
                     expand_star_imports=expand_star_imports,
                     remove_all_unused_imports=remove_all_unused_imports,
                     remove_duplicate_keys=remove_duplicate_keys,
-                    remove_unused_variables=remove_unused_variables))))
+                    remove_unused_variables=remove_unused_variables,
+                    ignore_init_module_imports=ignore_init_module_imports))))
```

Comment:
move the trailing `))))` to the next line, and add a trailing `,` on this line.

This list of args being passed is constantly growing, so it is a bit annoying that each addition results in two lines of change instead of only one.  You can prevent that from happening again by moving the `))))` to the next line.

Reply:
Fixed!